minard/templates/burst.html

```diff
@@ -0,0 +1,58 @@
+{% extends "layout.html" %}
+{% block title %}Burst Trigger{% endblock %}
+{% block head %}
+  {{ super() }}
+  <style>
+    .horizon .title
+  </style>
```

Comment:
What is this style block for?

Reply:
hmmm, not sure. Just 'borrowed' from one of the tellie pages. removed now.